src/mmrelay/migrate.py

```
@@ -0,0 +1,1871 @@
+                logger.warning("Failed to backup database: %s", e)
+
+        try:
+            if move:
+                logger.info("Moving database file: %s", db_path)
+                if dest.exists():
+                    dest.unlink()
+                shutil.move(str(db_path), str(dest))
+            else:
+                logger.info("Copying database file: %s", db_path)
+                shutil.copy2(str(db_path), str(dest))
+        except (OSError, IOError) as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)